### PR TITLE
Improve implementation of launchRequest

### PR DESCRIPTION
**Pull request by PhilippRo**
After some changes of node-sds and janus, it is possible to pass context names to the remote debugger.

This PR updates the node-sds version, and makes use of the new features to attach the right context.

See issue #18

Maybe fixes issue #17

### CODE REVIEW

**Comment by bkircher** [comment]
Checks fail.

@PhilippRo Can you rebase this PR.
@m-Schlitzer Can you review the changes and test if it works as expected/ I guess you need to build a new server for this.

**Comment by PhilippRo** [comment]
After some changes of node-sds and janus, it is possible to pass context names to the remote debugger.
This PR updates the node-sds version, and makes use of the new features to attach the right context.

See issue #18

File: src/debugSession.ts [hunk 1]
```diff
@@ -214,49 +220,19 @@ export class JanusDebugSession extends DebugSession {
...
-                                    this.sourceMap.addMapping(source, res.content.urls[0]);
-                                    resolve();
-                                } else if (res.content.urls.indexOf('JANUS') !== -1) {
-                                    this.sourceMap.addMapping(source, 'JANUS');
-                                    resolve();
-                                } else {
-                                    log.error(`launchRequest: could not find remote script that was just launched`);
-                                    reject(`Oops. Could not attach to script`);
-                                }
-                            } else {
```
Comment (bkircher) [inline src/debugSession.ts]: Thank you for removing this 👍

**Comment by PhilippRo** [comment]
All commits after c133f46f2dc120f8336b56840291d7568bedb66c need the newest version of jsrdbg running on your server.

**Comment by bkircher** [comment]
> The only time they don't work is when a script is launched via "Wartungsarbeit" in the privacyGuard client. The debugger extension terminates unexpectedly.

I'd ignore this and merge anyway. We can fix this issue afterwards.

**Comment by bkircher** [comment]
Cherry picked into c5d86dfe7d5e29e2fbb7b20bcff6422dce1dc8c2 (minus the changes in package-lock.json file).

Excellent work 👍 Thanks!

**Pull request closed without merging**